drivers/phy/marvell/phy-mvebu-cp110-comphy.c

```diff
@@ -714,6 +717,22 @@ static int mvebu_comphy_set_mode_10gbaser(struct phy *phy)
 	return mvebu_comphy_init_plls(lane);
 }
 
+/**
+ * Workaround for an HW eratta on the 260C BB. In some cases,
+ * some SFP+ transivers that are connected to another 260C won't
+ * be able to raise a link and get into an endless up/down loop.
+ * Writing this magic values works around this issue somehow. 
```

Comment:
Setting MVEBU_COMPHY_TX_EMPH_AMP_FORCE (bit 8) to lane register MVEBU_COMPHY_TX_AMP_EMP works around this issue (somehow?) maybe by forcing TX amplifier emphasis? (just guessing by abbreviations)

Reply:
We are not sure. Because the issue "Solves itself" even after we disable this bit. It probably just triggers some calibration. However, we have yet to get a proper response from Marvell.